airbyte-integrations/connectors/source-sendgrid/setup.py

```diff
@@ -31,6 +31,6 @@
     author="Airbyte",
     author_email="contact@airbyte.io",
     packages=find_packages(),
-    install_requires=["airbyte-protocol", "base-python", "backoff", "requests", "pytest==6.1.2"],
+    install_requires=["airbyte-cdk==0.1.3", "backoff", "requests", "pytest==6.1.2"],
```

Comment:
probably better to do something like `~=0.1` to pick up backwards compatible changes